.github/workflows/linter.yml

```diff
@@ -25,4 +25,4 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v1.56.2
\ No newline at end of file
+          version: v1.61
```

Comment:
After changing the go version the linter was failing, but with this version it worked